Anuradha Kumari.md

```diff
@@ -0,0 +1,39 @@
+Anuradha Kumari
+
+Photo: https://www.nicepng.com/png/detail/192-1927624_wolf-png-logo-graphic-free-stock-wolf-logo.png
+
+Github: https://github.com/AnuNancy
+
+Q. Remove Nth Node From End of List
+
+Answer-
+class Solution {
+public:
```

Comment:
You may want to separate the question from the solution file. Please refer to the `README.md` in the repository.